extensions/ql-vscode/src/common/commands.ts

```diff
@@ -185,7 +192,8 @@ export type EvalLogViewerCommands = {
   "codeQLEvalLogViewer.clear": () => Promise<void>;
 };
 
-export type AllCommands = BaseCommands &
+// All commands where the implementation is provided by this extension.
+export type AllCodeQLCommands = BaseCommands &
```

Comment:
Should this perhaps be called `AllExtensionCommands` or just `ExtensionCommands`? I believe we're mostly using extension to refer to the extension (like in `ExtensionApp`), rather than CodeQL (since this can also refer to the CLI).